src/utils.ts

```diff
@@ -209,6 +209,9 @@ export async function getPythPrice(splMint: PublicKey) {
   for (const symbol of data.symbols) {
     const price = data.productPrice.get(symbol)!;
     if (tokenToPythSymbol(splMintToToken(splMint)) == symbol) {
+      if (price == undefined){
+        return;
```

Comment:
why this?

Reply:
catching a case observed with mngo where price was returning undefined and breaking. Can't seem to reproduce now